presto-verifier/src/main/java/com/facebook/presto/verifier/framework/VerificationQueue.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.verifier.framework;
+
+public interface VerificationQueue
+{
+    boolean resubmit(Verification verification, QueryException e);
```

Comment:
This interface doesn't make a lot of sense to me. By reading the code I can see that you want a way to pass a streamlined way for callers to requeue queries, however, `VerificationQueue` is too generic a term for it. We can go either of the 2 ways:

1) Keep the class name and have 2 methods - `queue` and `requeue` or `submit` and `resubmit`
2) Change the class name to something like - `VerificationResubmitter` (naming is hard) to indicate that this is only for resubmits.

Reply:
I went for option 2 since 1 will make the privately used method `submit` publicly available.